src/sync/sync-utils.ts

```diff
@@ -1,15 +1,19 @@
 import {booleanFlag, BooleanFlags} from "../config/feature-flags";
 import RepoSyncState from "../models/reposyncstate";
-import logger from "../config/logger";
+import {getLogger} from "../config/logger";
 import {queues} from "../worker/queues";
 import sqsQueues from "../sqs/queues";
 import Subscription from "../models/subscription";
+import {LoggerWithTarget} from "probot/lib/wrap-logger";
+import Logger from 'bunyan';
 
 
 export async function findOrStartSync(
 	subscription: Subscription,
+	logger: LoggerWithTarget | Logger,
```

Comment:
If you're adding a new optional param, add it to the end.  You don't need to do the `logger = logger || getLogger("sync-utils");` below, you can just do `logger = getLogger("sync-utils")`

Reply:
Good catch, it is actually not optional, it is required. So I'll just remove the line